### ISSUE TITLE
Put solving algorithms into separate file

### ISSUE THREAD

**Issue by OliverMatthews** [OliverMatthews/rubiksCube#1]
Put totalSolve() and all subsidiary functions into a separate file.

**Comment by OliverMatthews** [comment]
Placed on hold, all these functions need to be altered to have inputs rather than directly editing the game cube.

**Comment by OliverMatthews** [comment]
Raised issue #11 which must be completed before this issue can continue work.

**Comment by OliverMatthews** [comment]
Done in #12
